### PR TITLE
Stripe checkout sessions, success/cancel page, subscribed user -> home

### PR DESCRIPTION
**Pull request by andrewalson**
Current state of payments flow presents user with working checkout sessions for Basic & Pro plans, success & thank you OR cancellation/could not complete page, and routes them back to home for now. Configured ESLint to fix Vercel warnings. 

Test Stripe payment portals in Dev mode with test cards from: [https://docs.stripe.com/testing#cards](https://docs.stripe.com/testing#cards)

### CODE REVIEW

**Review by tanmay-sketch — APPROVED** [review]
Changes look good

File: package.json
Comment (saikiransomanagoudar) [inline package.json:1]: Looks good.

File: package-lock.json
Comment (saikiransomanagoudar) [inline package-lock.json:1]: Looks good.

**Review by vamsi2792 — APPROVED** [review]
Changes works well.

**Review by saikiransomanagoudar — COMMENTED** [review]
Looks good

**Comment by saikiransomanagoudar** [comment]
Please merge it to the main.

**Pull request merged**